arrow/arrow.py

```diff
@@ -56,6 +57,12 @@ class Arrow(object):
     _ATTRS = ["year", "month", "day", "hour", "minute", "second", "microsecond"]
     _ATTRS_PLURAL = ["{}s".format(a) for a in _ATTRS]
     _MONTHS_PER_QUARTER = 3
+    _SECS_PER_MINUTE = float(60)
+    _SECS_PER_HOUR = float(3600)  # 60 * 60
```

Comment:
Any reason why you expanded the multiplied values and wrote them in a comment? May be more clear to simply keep the multiplication in place inside of the cast.

Reply:
I'll expand them in the next commit for clarity